freeipa_community_portal/config.py

```diff
@@ -27,7 +27,7 @@
 class Config(object):
     development_config = ('freeipa_community_portal/conf/'
                           'freeipa_community_portal_dev.ini')
-    deployment_config = '/etc/freeipa_community_portal.ini'
+    deployment_config = '/etc/ipa/extensions/community-portal/portal.ini'
```

Comment:
These same string literals appear in freeipa-portal-install and create-portal-user.

We should define them here in the lib and import from the scripts.